code/modules/client/verbs/pointing_cooldown.dm

```diff
@@ -0,0 +1,26 @@
+#define POINT_STACK_LIMIT 3
+/client
+	/// Your clients allowed pointing stacks.
+	var/point_stacks = POINT_STACK_LIMIT
```

Comment:
making this var on every client is going to add pointless overhead.

Reply:
what do otherwise?

Reply:
> making this var on every client is going to add pointless overhead.

> pointless

ba dum tish

Reply:
> making this var on every client is going to add pointless overhead.

ok but how else are you going to track something on a per client basis